lib/components/page.dart

```diff
@@ -41,22 +40,26 @@ const kItemsToFetchForPagination = 20;
 class BccmPage extends ConsumerStatefulWidget {
   final Future<Query$Page$page> pageFuture;
   final Future Function({bool? retry}) onRefresh;
+  final ScrollController? scrollController;
 
   const BccmPage({
     super.key,
     required this.pageFuture,
     required this.onRefresh,
+    this.scrollController,
   });
 
   @override
-  ConsumerState<BccmPage> createState() => _BccmPageState();
+  ConsumerState<BccmPage> createState() => _BccmPageState(scrollController ?? ScrollController());
 }
 
 class _BccmPageState extends ConsumerState<BccmPage> {
   GlobalKey<State<FutureBuilder<Query$Page$page>>> futureBuilderKey = GlobalKey();
   Map<String, PaginationStatus<Fragment$ItemSectionItem>> paginationMap = {};
-  ScrollController scrollController = ScrollController();
   bool loadingBottomSectionItems = false;
+  final ScrollController scrollController;
+
+  _BccmPageState(this.scrollController);
```

Comment:
its a nitpick but I still think using initState is better than constructors, even when it's unecessary, just to simplify/streamline how we do initialization. I'll approve this anyway though.

Reply:
> its a nitpick but I still think using initState is better than constructors, even when it's unecessary, just to simplify/streamline how we do initialization. I'll approve this anyway though.

Sure. Was trying to avoid late variable.